app-tracking-protection/vpn-impl/src/main/res/layout/row_exclusion_list_app.xml

```diff
@@ -59,14 +55,15 @@
         app:layout_constraintStart_toStartOf="@+id/deviceShieldAppEntryName"
         app:layout_constraintTop_toBottomOf="@+id/deviceShieldAppEntryName"
         app:textType="secondary"
-        app:typography="caption"
+        app:typography="body2"
```

Comment:
Is this change expected? Here it seems like the reasoning uses `caption` https://www.figma.com/file/BOHDESHODUXK7wSRNBOHdu/%F0%9F%A4%96-Android-Components?type=design&node-id=6570-44195&mode=design&t=E0bSIjWvBADfdqPx-0

Reply:
ditto